### DISCUSSION TITLE
how to extend tailwind config file's settings?

### DISCUSSION THREAD

**Discussion by mrl24730** [tailwindlabs/tailwindcss#5395] [Help]
I have a CSS design system built with tailwind css. how to extend that `tailwind.config.js`?

Similar to the below sample, but import those all-in-one instead spreading those one by one

```js
const defaultTheme = require('tailwindcss/defaultTheme')

fontFamily: {
  sans: ['rebot', ...defaultTheme.fontFamily.sans], 
},
```

**Comment by dance2die** [comment]
One thing I can think of is to create a [preset](https://tailwindcss.com/docs/presets) and do

```js
module.exports = {
  presets: [
    require('@org/design-system')
  ],
...
```

It'd still require adding fonts/colors etc manually though.  
But at least you can apply it in multiple projects
